get_social_media_handles.py

```diff
@@ -0,0 +1,45 @@
+# Works only if urls are present in HTML Content
+import requests
+import re
+
+def get_social_media_handles(url):
+    """Get social media handles from a webpage.
+    
+    Args:
+        url (str): The URL of the webpage.
+    
+    Returns:
+        dict: A dictionary containing social media handles for each platform.
+    """
+    # Local function to generate regex pattern for social media URLs
+    def generate_regex_pattern():
+        """Generate regex pattern for social media URLs.
+        
+        Returns:
+            str: The regex pattern for social media URLs.
+        """
+        return r'(https?://(?:www\.)?(facebook\.com|twitter\.com|instagram\.com|linkedin\.com)/[^/?"]+)'
```

Comment:
we have hardcoded these - its a good idea to put these into a constant so if we have to add more later we dont have to edit the regex